Client/Ecosia/Extensions/HomepageViewController+Ecosia.swift

```diff
@@ -92,14 +78,35 @@ extension HomepageViewController: NTPLibraryDelegate {
     }
 }
 
-extension HomepageViewController: YourImpactDelegate {
-    func yourImpact(didSelectURL url: URL) {
-        dismiss(animated: true)
-        homePanelDelegate?.homePanel(didSelectURL: url, visitType: .link, isGoogleTopSite: false)
+extension HomepageViewController: NTPImpactCellDelegate {
+    func impactCellButtonAction(info: ClimateImpactInfo) {
```

Comment:
You are right, this is the action of the button inside the impact cell, but I guess it would be clearer something like `impactCellButtonClick(withInfo info: ClimateImpactInfo)`?

Reply:
I lean on something more declarative like:
`impactCellButtonClickedWithInfo(_ info: ClimateImpactInfo)` (clicked or tapped your preference 👍 )